README.md

```diff
@@ -42,6 +42,11 @@ php composer.phar require chobie/jira-api-restclient ^2.0@dev
 
 * [Composer](https://getcomposer.org/download/)
 
+## Authentication
+
+This client uses basic authentication on the Jira Cloud APIs. Basic authentication requests for Jira Cloud should use [API tokens](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) instead of Atlassian account passwords, as support for basic authentication with passwords has been [deprecated](https://community.developer.atlassian.com/t/announcement-deprecation-of-basic-authentication-with-passwords-and-cookie-based-authentication-in-jira-cloud-rest-apis/15687).
```

Comment:
Usage of `Jira Cloud` term alone is incorrect. Better replace `on the Jira Cloud APIs` with `on the Jira Instance (standalone or in the cloud)`.